calculator.py

```diff
@@ -108,10 +123,14 @@ def divide(self,a: float,b: float) -> float:
 
 
 def start_calculator():
+    t = threading.Thread(target=thread_additions,)
+    t.start()
+    cal= Calculator()
     args = create_parser()
     salusion=take_expresion(args.f)
-    cal= Calculator()
     print(cal.calculate(salusion))
+    print(f" Total operations: {cal.add.count_operation + cal.subtract.count_operation + cal.multiply.count_operation + cal.divide.count_operation} \n Add operations: {cal.add.count_operation} \n Subtract operations: {cal.subtract.count_operation} \n Multiply operations: {cal.multiply.count_operation} \n Divide operations: {cal.divide.count_operation} \n") # pylint: disable=no-member
+    t.do_run = False
```

Comment:
Why do you print in a background thread? This belongs to a later task.
Anyway, if you do this here:

Try to do this without setting an attribute on the current thread object.
You can create a class, that has the `thread_additions` method (also, find a better name than this) and a `running` member and check it in the loop.